packages/ui/utils/abbreviator.tsx

```diff
@@ -0,0 +1,15 @@
+import { Text } from '@chakra-ui/react';
+import { ReactNode } from 'react';
+
+import { PopoverTooltip } from '@ui/components/shared/PopoverTooltip';
+
+export function abbreviator(str: string | undefined, len: number): string | undefined | ReactNode {
```

Comment:
```suggestion
export function abbreviator(str?: string, len: number): string | undefined | ReactNode {
```

Reply:
I think if it's either the value or undefined using the `?` optional operator is a bit cleaner in my opinion. But we could keep it as is. ✌️